docs/UPDATING_RUNTIMES.md

```diff
@@ -0,0 +1,10 @@
+# Updating runtimes
+
+This project uses `Erlang`, `Elixir` and `Javascript` (NodeJS). We should keep these updated often, to reduce vulnerabilities and keep the project in tip top shape.
+
+To update, simply alter the versions in the following files:
+- `.github/workflows/ci.yml` - just `Erlang` and `Elixir`
+- `.tool-version` - this is used `asdf-vm` which we recommend everyone uses
```

Comment:
```suggestion
- `.tool-version` - this is used by `asdf-vm`, which is the tool that we recommend everyone uses to manage runtime versions
```

Reply:
I was going to run grammarly on it, but thanks! XD